### DISCUSSION TITLE
Cant connect to the api

### DISCUSSION THREAD

**Discussion by JeshMate** [louisgv/local.ai#113] [Q&A]
I cant seem to ping the api whatsoever apart from getting a pong html page, even when posting to /v1/completions i just get a refused connection regardless if im using localhost or a public ip.

**Comment by louisgv** [comment]
Did you try the CURL example? https://github.com/louisgv/local.ai/wiki/API-with-cURL

**Reply by JeshMate** [reply]
yeah i did, it idles for a little then fails to connect, locally and remotely, sometimes it does work though and when it does the output is expected. the main goal is i wanted to use axios to send posts but it doesnt like it
